hack/cleanup.sh

```diff
@@ -15,41 +15,30 @@ sudo bash -c "
     set +e
     systemctl stop --now microshift 2>/dev/null
     systemctl disable microshift 2>/dev/null
-    systemctl stop --now microshift-containerized 2>/dev/null
-    systemctl disable microshift-containerized 2>/dev/null
-    podman stop microshift 2>/dev/null
-    podman stop microshift-aio 2>/dev/null
 
     echo Removing non-OVN crio pods
-    
-    crictl pods | tail -n +2 | grep -vE openshift-ovn-kubernetes | awk '{print \$1}' | xargs crictl stopp
-    crictl pods | tail -n +2 | grep -vE openshift-ovn-kubernetes | awk '{print \$1}' | xargs crictl rmp
-    
+    NOVN_PODS=\$(crictl pods | tail -n +2 | grep -vE openshift-ovn-kubernetes | awk '{print \$1}')
+    [ ! -z \"\${NOVN_PODS}\" ] && crictl stopp \${NOVN_PODS}
+    [ ! -z \"\${NOVN_PODS}\" ] && crictl rmp   \${NOVN_PODS}
 
     echo Removing all crio pods
     until crictl rmp --all --force 1>/dev/null; do sleep 1; done
 
-    echo Removing crio containers
-    crictl rm --all --force 1>/dev/null
-
-    echo Removing crio images
-    crictl rmi --all --prune 1>/dev/null
+    echo Removing crio container and image storage
+    crio wipe -f &>/dev/null
+    systemctl restart crio
 
-    echo Killing conmon, pause processes
+    echo Killing conmon, pause and OVN processes
+    ovs-vsctl del-br br-int
     pkill -9 conmon
     pkill -9 pause
     pkill -9 ovn-controller
     pkill -9 ovn-northd
     pkill -9 ovsdb-server
 
-    echo Removing /var/lib/microshift
-    crio wipe -f
-    systemctl restart crio
-    echo Reverting microshift-ovs-init.service configuration
-    /usr/bin/configure-ovs.sh OpenShiftSDN
+    echo Removing MicroShift and OVN configuration
```

Comment:
@mangelajo 
There is no /var/lib/ovn directory, but /var/lib/ovnk - do we want the latter removed?